src/test/java/com/kodekonveyor/repo/api/ElementDTOTestData.java

```diff
@@ -0,0 +1,73 @@
+package com.kodekonveyor.repo.api;
+
+
+public class ElementDTOTestData {
+
+  public static final String SYNTAX_ID = "SyntaxID";
+  public static final String FOUND_LOCAL_ID = "FOUND_LOCAL";
+  public static final String FOUND_SERVER_ID = "FOUND_SERVER";
+  public static final String REMOTE_ID = "REMOTE";
+  public static final String NOT_FOUND_LOCAL_ID = "NOT_FOUND_LOCAL";
+  public static final String NOT_FOUND_SERVER_ID = "NOT_FOUND_SERVER";
+
+
+
+  public static ElementDTO get() {
+    final ElementDTO element = new ElementDTO();
+    element.setCommit("foundLocalCommit");
+    element.setSource("sourceFoundLocal");
+    element.setTarget("targetFoundLocal");
+    element.setId(FOUND_LOCAL_ID);

```

Comment:
Code duplication. Create a private method for building ElementDTO
```
private ElementDTO buildElementDTO(
        final String id,
        final String commit,
        final String source,
        final String target
    ) {
    final ElementDTO element = new ElementDTO();

    element.setId(id);
    element.setCommit(commit);
    element.setSource(source);
    element.setTarget(target);

    return element;
}
```
And replace body of each get* method to a call of this method with the right parameters.